hexkit/__init__.py

```diff
@@ -15,4 +15,4 @@
 
 """A Toolkit for Building Microservices using the Hexagonal Architecture"""
 
-__version__ = "0.9.3"
+__version__ = "0.9.4"
```

Comment:
This is a new feature, thus maybe bumpt the minor:
```suggestion
__version__ = "0.10.0"
```